### DISCUSSION TITLE
how to accept line or word?

### DISCUSSION THREAD

**Discussion by azinsharaf** [zbirenbaum/copilot.lua#281] [Q&A]
when copilot suggests a block of code how can i just accept some lines of it? Pressing Tab accepts all of it. I like to have a way to accept the suggestion line by line. I've tried `accept_word` and `accept_line` and set them to `true' but when i do it it stops auto suggestion.

**Comment by MattiasEXE** [comment]
I have the same question, did you have any solutions?

**Comment by AntoineGS** [comment]
While I have not used this myself, the `accept_word` and `accept_line` are part of the `keymap` configuration, and should therefore contain the keyboard key combination to trigger these two actions (instead of `true`).
If this does not work please open an Issue and I will look into it further 🙂

**Comment by MattiasEXE** [comment]
I did try to use the `keymap` configurations as you said, but that didn't work. After some looking around I think this has to do with me using blink.cmp for the suggestions (default [LazyExtra config](https://www.lazyvim.org/extras/ai/copilot)).
